scheduler/src/cook/mesos/api.clj

```diff
@@ -274,6 +440,21 @@
 ;;;            "cpus": 1.5,
 ;;;            "mem": 1000}]}
 ;;;
+;;; With a docker container ...
+;;; {"jobs": [{
+;;;  "command" : "/bin/echo $MYENV Hello World"
+;;;  "uuid"  : ""6e0eeb86-8c93-11e5-adfd-f8b156b232c9""
+;;;   "max-retries" : 3
+;;;   "uris" : [{"value" : "file:///home/sdegler/0.pref"}]
+;;;   "env" : "MYENV abc"
+;;;   "container" : { "type" : "docker",
+;;;                   "docker" : { "image" : "bp4_res_20150904.patched" ,
+;;;                                "parameters" : [ {"env" : "KRB5CCNAME=${KRB5CCNAME}" }, {"user" : "10138"}],
```

Comment:
its getting passed to the docker command line itself, the  example is from what I had been doing with marathon.
It is not necessarily needed, but would always be available.